example/exampleTags/plugin.js

```diff
@@ -23,8 +23,12 @@ import availableTags from './tags.json';
 /**
  * @returns {function} The plugin install function
  */
-export default function exampleTagsPlugin() {
+export default function exampleTagsPlugin(namespaceToSaveAnnotations) {
```

Comment:
Should we make this an `options` object instead, in case we would need to add more options later? I think most of our configurable plugins follow this pattern. We can then add a JSDoc `@typedef` above to define the available options.